docs/page.css

```diff
@@ -186,7 +186,7 @@ table td:first-child {
 	padding-left: 0;
 }
 
-code:not(.inline) {
+code:not(.inline, [inline]) {
```

Comment:
Small update for convenience, allows writing `<code inline>...</code>` instead of `<code class="inline">...</code>`.